src/components/ClusterManager.js

```diff
@@ -206,12 +206,15 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     refreshClusters: PropTypes.func.isRequired
   }
 
+  static defaultProps = {
+    canCompute: false
```

Comment:
nit: Technically this is required, and it would be slightly more correct to tweak the logic at the caller to always produce a bool.